### ISSUE TITLE
Connect Video no longer opens

### ISSUE THREAD

**Issue by benhalpern** [thepracticaldev/dev.to#8749]
This is not a _core_ feature so it's not the end of the world, but worth noting that video functionality is not currently working as expected.

Reproduce:

Go to Connect -> type `/call` -> Check console.

**Comment by rhymes** [comment]
Thanks for the issue! We'll take your request into consideration and follow up if we decide to tackle this issue.

To our amazing contributors: issues labeled `type: bug` are always up for grabs, but for feature requests, please wait until we add a `ready for dev` before starting to work on it.

To claim an issue to work on, please leave a comment. If you've claimed the issue and need help, please ping @thepracticaldev/oss and we will follow up within 3 business days.

For full info on how to contribute, please check out our [contributors guide](https://github.com/thepracticaldev/dev.to/blob/master/CONTRIBUTING.md).

**Comment by narender2031** [comment]
@benhalpern & @rhymes. This issue is fixed in another PR. We can close this issue.
